hcl/writer.go

```diff
@@ -465,9 +465,21 @@ func writeTuple(pbody, body *hclwrite.Body, attr string, value cty.Value) {
 }
 
 // Interpolate replaces the hardcoded resources link
-// with TF interpolation
+// with TF interpolation.
+// I has the following format:
+// * VALUE => ${aws_instance.name.attribute}
+// So then each value matching the VALUE will
+// be replaced with the reference
 func (w *Writer) Interpolate(i map[string]string) {
-	if !w.opts.Interpolate {
+	// If Interpolation is disabled or
+	// a module without specific attributes to
+	// create as variables (so all are variables),
```

Comment:
Meaning no call to a module? And simple `variable` block?

Reply:
If you have a `--module my-module` without the `--module-variables` it would replace all values for variables, so no interpolation needed as everything would be a variable.